services/openvpn/session/validator_mocks.go

```diff
@@ -45,7 +45,7 @@ func (manager *MockSessionManager) Create(peerID identity.Identity) (sessionInst
 	return session.Session{}, nil
 }
 
-// FindSession returns mocked session
+// FindClientSession returns mocked session
 func (manager *MockSessionManager) FindSession(sessionID session.SessionID) (session.Session, bool) {
```

Comment:
Comment and function name does not match.

Reply:
Fixed